tests/lib/snaps/store/test-snapd-refresh-control-provider.v2/meta/snap.yaml

```diff
@@ -0,0 +1,17 @@
+name: test-snapd-refresh-control-provider
```

Comment:
Same consideration described for the previous snap

Reply:
Done.